corehq/apps/api/resources/v0_4.py

```diff
@@ -120,6 +120,7 @@ def obj_get_list(self, bundle, domain, **kwargs):
 
         def wrapper(doc):
             if doc['doc_type'] in xform_doc_types:
+                doc.pop('user_id', None)  # needed when xform is stored in sql
                 return xform_doc_types[doc['doc_type']].wrap(doc)
```

Comment:
Who owns this issue at the moment?  Is it the case that the forms api doesn't work at all for sql domains?  Is anyone working on a new API for SQL backed forms?